configure.py

```diff
@@ -444,8 +444,6 @@ def process_command_line(args): # pylint: disable=too-many-locals
 
     install_group.add_option('--prefix', metavar='DIR',
                              help='set the install prefix')
-    install_group.add_option('--destdir', metavar='DIR',
```

Comment:
I'd be very happy if the destdir variable could be removed from the configure script (modulo error message) as the mixing of prefix and destdir has always been great brainfuck. Being able to push this to install is of great value in my opinion.